web/client/reducers/featuregrid.js

```diff
@@ -346,7 +347,8 @@ function featuregrid(state = emptyResultsState, action) {
             deleteConfirm: false,
             drawing: false,
             newFeatures: [],
-            changes: []
+            changes: [],
+            closer: action.closer
```

Comment:
not sure about this prop added here

Reply:
it is needed to set closer with QueryPanel in case of close featuregrid temporarily to apply some query before return to the feature grid again.